README.md

```diff
@@ -117,33 +118,36 @@ Bmg.sequel(:suppliers, sequel_db)
 #   {:array=>false})
 ```
 
-### Reading files (csv, Excel, text)
+### Reading data files (json, csv, yaml, text, xls & xlsx)
 
 Bmg provides simple adapters to read files and reach Relationland as soon as
 possible.
 
-#### CSV files
+#### JSON files
 
 ```ruby
-csv_options = { col_sep: ",", quote_char: '"' }
-r = Bmg.csv("path/to/a/file.csv", csv_options)
+r = Bmg.json("path/to/a/file.json")
 ```
 
-Options are directly transmitted to `::CSV.new`, check Ruby's standard
-library.
+The json file is expected to contain tuples of same heading.
 
-#### Excel files
+#### YAML files
 
-You will need to add [`roo`](https://github.com/roo-rb/roo) to your Gemfile to
-read `.xls` and `.xlsx` files with Bmg.
+```ruby
+r = Bmg.yaml("path/to/a/file.yaml")
+```
+
+The yaml file is expected to contain tuples of same heading.
+
+#### CSV files
 
 ```ruby
-roo_options = { skip: 1 }
-r = Bmg.excel("path/to/a/file.xls", roo_options)
+csv_options = { col_sep: ",", quote_char: '"' }
+r = Bmg.csv("path/to/a/file.csv", csv_options)
 ```
 
-Options are directly transmitted to `Roo::Spreadsheet.open`, check roo's
-documentation.
+Options are directly transmitted to `::CSV.new`, check Ruby's standard
```

Comment:
Possible improvement: add the `headers` flag.

https://ruby-doc.org/stdlib-2.6.1/libdoc/csv/rdoc/CSV.html#class-CSV-label-CSV+with+headers